compiler/luci/import/include/luci/Import/CircleReader.h

```diff
@@ -146,7 +148,7 @@ class CircleReader
   CircleOperators native_operators() const { return wrap(_native_subgraph->operators()); }
   VectorWrapper<int32_t> native_inputs() const { return wrap(_native_subgraph->inputs()); }
   VectorWrapper<int32_t> native_outputs() const { return wrap(_native_subgraph->outputs()); }
-  std::string native_name() const { return _native_subgraph->name()->str(); }
+  std::string native_name() const { return fb_string2std_string(_native_subgraph->name()); }
```

Comment:
I would like to introduce this changes via helper function, but it's possible to make `nullptr` check here.